Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -110,7 +113,11 @@ click: function () { closeClicked.raiseEvent(this); }');
             that._descriptionSubscription = subscribeAndEvaluate(viewModel, 'description', function(value) {
                 // Set the frame to small height, force vertical scroll bar to appear, and text to wrap accordingly.
                 frame.style.height = '5px';
-                frameContent.innerHTML = value;
+                if (value.indexOf('<script') > -1 && that.allowScripts) {
```

Comment:
There's some duplicate logic here and in `stringToHtml`. You already check for the script and set `frameContent.innerHTML`. Can you clean that up?